src/queries/getPrimaryKeys.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../db')
```

Comment:
NOTE: I need to remove this file, it's not actually used. Was a part of an early, different attempt to get constraints... but `getConstraints.js` above provided everything needed.